dev-java/fec/fec-1.0.4-r4.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
we might want to migrate it to eapi 8

Reply:
> we might want to migrate it to eapi 8

Of course, changed it and also removed a superfluous line.